dashboard/test/models/authentication_option_test.rb

```diff
@@ -54,8 +55,8 @@ class AuthenticationOptionTest < ActiveSupport::TestCase
   test 'user can have multiple authentication options' do
     assert_creates(User) do
       user = create(:user, :with_google_authentication_option, :with_clever_authentication_option)
-      assert user.authentication_options.length == 2
-      assert user.authentication_options.first.hashed_email == user.hashed_email
+      assert_equal 3, user.authentication_options.length
+      assert_equal user.hashed_email, user.authentication_options.first.hashed_email
```

Comment:
Why do we expect the user to have 3 authentication options now instead of 2?

Reply:
the two `with_authentication_option` traits each add an authentication option on top of the one that's created by migrating a standard user. In several places, I resolved this by creating the user with SSO properties directly (which would then be transformed into the appropriate authentication option), but in this case I would have had to do that with only one of the desired auth options. I decided to leave them both as "extras" for parallelism.